scanners/ios/appclip.go

```diff
@@ -0,0 +1,36 @@
+package ios
+
+import (
+	"github.com/bitrise-io/bitrise-init/models"
+	"github.com/bitrise-io/bitrise-init/steps"
+	envmanModels "github.com/bitrise-io/envman/models"
+	"github.com/bitrise-io/go-xcode/xcodeproj"
+)
+
+func schemeHasAppClipTarget(scheme xcodeproj.SchemeModel, targets []xcodeproj.TargetModel) bool {
```

Comment:
NOTE: we have an issue with detecting app-clips, not sure if this code has the same bug, but worth checking: https://bitrise.atlassian.net/browse/STEP-1754

Reply:
It looks to me that the code was moved here from utility.go without changes (except *appendExportAppClipStep*), would keep the bugfix in a followup PR to limit the scope.